X244511

raul-lopez-telus opened the pull request:
### Sumary
Delivering Netcracker Configurations - creating a patch - x244511
### Details
added 7 object_types and 10 objects

RJaco1 committed:
Delivering Netcracker Configurations - creating a patch - x244511

RJaco1 committed:
Delivering Netcracker Configurations - creating a patch - x244511

henry-molina-telus commented:
Hi @raul-lopez-telus 

Please, apply the following corrections:

1. Remove the Change Request configuration file from the repository (BE/COM/com_common/scm/com/telus/dsu/nc_objects/obj_9162019460013712668.xml). This is not usually versioned.
2. Add me as a reviewer of your pull request.


<img width="248" alt="image" src="https://user-images.githubusercontent.com/52331453/142634087-9d28faf5-4a0a-43ba-b66b-3345dbac773f.png">

RJaco1 committed:
removed object file 9162019460013712668 as it is not usually versioned

raul-lopez-telus commented:
> Hi @raul-lopez-telus
> 
> Please, apply the following corrections:
> 
> 1. Remove the Change Request configuration file from the repository (BE/COM/com_common/scm/com/telus/dsu/nc_objects/obj_9162019460013712668.xml). This is not usually versioned.
> 2. Add me as a reviewer of your pull request.
> 
> <img alt="image" width="248" src="https://user-images.githubusercontent.com/52331453/142634087-9d28faf5-4a0a-43ba-b66b-3345dbac773f.png">

Hi @henry-molina-telus

Appreciate your feedback, I have removed the change request configuration file from the repository and I have added you as a reviewer as instructed.

Best regards.

henry-molina-telus commented:
Thank you so much, @raul-lopez-telus.

raul-lopez-telus commented:
> Thank you so much, @raul-lopez-telus.
